tests/pipelines/animatediff/test_animatediff.py

```diff
@@ -126,6 +131,15 @@ def get_dummy_inputs(self, device, seed=0):
         }
         return inputs
 
+    def test_from_pipe_consistent_config(self):
+        # Note: This test tries to load components from the "hf-internal-testing/tiny-stable-diffusion-pipe" or
+        # "hf-internal-testing/tiny-stable-diffusion-xl-pipe" depending upon the AnimateDiff pipeline being
+        # SD1.5 or SDXL. The dummy pipe components have different initialization configs than required here.
+        # For example, cross_atttention_dim is 32 for dummy model whereas we are using 8 here to speed up tests.
+        # I think we might need to push dummy models with the initialization configuration used here and rewrite
+        # the test to use that variant for this to pass. We skip it for now.
+        pass
```

Comment:
Why not implement it in this PR itself?

Reply:
@sayakpaul Could you review this again? If all is well, we'd need to move `a-r-r-o-w/tinier-stable-diffusion-pipe` to `hf-internal-testing` org.

Also need help with getting the correct cpu expected slices for the failing tests: `test_ip_adapter_single` and `test_dict_tuple_outputs_equivalent`.